### PR TITLE
changepassword update

### PR DESCRIPTION
**Pull request by avocadosec**
Instructions for using manage.py to change scantron user password.

### CODE REVIEW

**Comment by avocadosec** [comment]
@derpadoo could you please review this?

File: README.md [hunk 1]
```diff
@@ -157,6 +157,14 @@ pip3 install passlib
 python3 -c "from passlib.hash import sha512_crypt; import getpass; print(sha512_crypt.encrypt(getpass.getpass()))"
 ```
 
+#### Change scantron user password with manage.py (optional)
+
+Change into the master directory `scantron/master` and run the following to change the scantron user password
```
Comment (derpadoo) [inline README.md]: Change to:

```
cd into the master directory `scantron/master` and run the following to change the `scantron` (or whatever user needs their password changed) user password.
```

File: ansible-playbooks/roles/master/tasks/main.yml [hunk 2]
```diff
@@ -213,7 +213,7 @@
   shell: echo "from django.contrib.auth.models import User; User.objects.create_user('{{ django_user }}', '{{ django_user_email }}', '{{ scantron_secrets['django_user_password']  }}')" | {{ venv_python }} {{ scantron_dir }}/manage.py shell
   args:
     chdir: "{{ scantron_dir }}"
-  ignore_errors: yes  # Only applicable if playbook has alread been run.  Use 'python manage.py change changepassword scantron'
+  ignore_errors: yes  # Use 'python manage.py changepassword scantron'
```
Comment (derpadoo) [inline ansible-playbooks/roles/master/tasks/main.yml]: Change to:

```
# Only applicable if playbook has alread been run.  Use 'python manage.py changepassword scantron'.
```

Comment (derpadoo) [reply ansible-playbooks/roles/master/tasks/main.yml]: Thanks for the contribution @avocadosec !

File: README.md (hunk 1 above)
Comment (avocadosec) [reply README.md]: Changed

File: ansible-playbooks/roles/master/tasks/main.yml (hunk 2 above)
Comment (avocadosec) [reply ansible-playbooks/roles/master/tasks/main.yml]: Changed

**Pull request merged**